tests/modules/object-cache/persistent-object-cache-health-check/health-check-test.php

```diff
@@ -0,0 +1,132 @@
+			function ( $thresholds ) {
+				return array_merge( $thresholds, array( 'comments_count' => 0 ) );
+			}
+		);
+
+		$result = perflab_oc_health_should_persistent_object_cache();
+
+		$this->assertTrue( $result );
+
+		remove_all_filters( 'perflab_oc_site_status_persistent_object_cache_thresholds' );
+	}
+
+	function test_object_cache_posts_threshold() {
+		add_filter(
+			'perflab_oc_site_status_persistent_object_cache_thresholds',
+			function ( $thresholds ) {
+				return array_merge( $thresholds, array( 'posts_count' => 0 ) );
+			}
+		);
+
+		$result = perflab_oc_health_should_persistent_object_cache();
+
+		$this->assertTrue( $result );
+
+		remove_all_filters( 'perflab_oc_site_status_persistent_object_cache_thresholds' );
+	}
+
+	function test_object_cache_terms_threshold() {
+		add_filter(
+			'perflab_oc_site_status_persistent_object_cache_thresholds',
+			function ( $thresholds ) {
+				return array_merge( $thresholds, array( 'terms_count' => 1 ) );
+			}
+		);
+
+		$result = perflab_oc_health_should_persistent_object_cache();
+
+		$this->assertTrue( $result );
+
+		remove_all_filters( 'perflab_oc_site_status_persistent_object_cache_thresholds' );
+	}
+
+	function test_object_cache_options_threshold() {
+		add_filter(
+			'perflab_oc_site_status_persistent_object_cache_thresholds',
+			function ( $thresholds ) {
+				return array_merge( $thresholds, array( 'options_count' => 100 ) );
+			}
+		);
+
+		$result = perflab_oc_health_should_persistent_object_cache();
+
+		$this->assertTrue( $result );
+
+		remove_all_filters( 'perflab_oc_site_status_persistent_object_cache_thresholds' );
+	}
+
+	function test_object_cache_users_threshold() {
+		add_filter(
+			'perflab_oc_site_status_persistent_object_cache_thresholds',
+			function ( $thresholds ) {
+				return array_merge( $thresholds, array( 'users_count' => 0 ) );
+			}
+		);
+
+		$result = perflab_oc_health_should_persistent_object_cache();
+
+		$this->assertTrue( $result );
+
+		remove_all_filters( 'perflab_oc_site_status_persistent_object_cache_thresholds' );
+	}
+
+	function test_object_cache_alloptions_count_threshold() {
+		add_filter(
+			'perflab_oc_site_status_persistent_object_cache_thresholds',
+			function ( $thresholds ) {
+				return array_merge( $thresholds, array( 'alloptions_count' => 100 ) );
+			}
+		);
+
+		$result = perflab_oc_health_should_persistent_object_cache();
+
+		$this->assertTrue( $result );
+
+		remove_all_filters( 'perflab_oc_site_status_persistent_object_cache_thresholds' );
+	}
+
+	function test_object_cache_alloptions_bytes_threshold() {
+		add_filter(
+			'perflab_oc_site_status_persistent_object_cache_thresholds',
+			function ( $thresholds ) {
+				return array_merge( $thresholds, array( 'alloptions_bytes' => 1000 ) );
+			}
+		);
+
+		$result = perflab_oc_health_should_persistent_object_cache();
+
+		$this->assertTrue( $result );
+
+		remove_all_filters( 'perflab_oc_site_status_persistent_object_cache_thresholds' );
+	}
```

Comment:
Not critical, but all of these could be replaced with a single test method with parameters and an associated`@dataProvider` method.

There's a ton of duplicate code here that we could get rid of by doing that.